### DISCUSSION TITLE
Guidelines for third-party backends?

### DISCUSSION THREAD

**Discussion by hooverdc** [RealOrangeOne/django-tasks#107] [General]
I'd like to hear everyone's opinions on how to implement a third-party backend for Django tasks. Let's take an SQS backend for example. I would name it `django_tasks_sqs`, and other backends could follow that pattern. Are there any guidelines for backends and workers that should be followed?

**Comment by RealOrangeOne** [comment]
I personally don't see the need to be too prescriptive, especially on package name. Package discoverability relies on a lot more than just its name.

So long as the package meets the API spec, and works, I'm happy. If you do find issues, or APIs which don't feel quite right, please report back!

Using SQS is presumably just acting as the queue _store_, therefore there's going to be a lot of duplication of the code from say the `db_worker` management command. This is something I'd not considered, so I've made a note of it for future: https://github.com/RealOrangeOne/django-tasks/discussions/7#discussioncomment-10538041
